Add firewall details and clarify post-install config

jrwdunham opened the pull request:
- Adds a firewall section to the Ubuntu packages installation page (install-ubuntu.rst.)
- Clarifies the "Post-install configuration" sections of the install-ubuntu.rst, install-centos.rst, and install-ansible.rst pages.
- Adds more details to the "Firewall" section of install-advanced.rst.

Addresses https://github.com/artefactual/archivematica/issues/954

sevein commented on admin-manual/installation/installation.rst:
```diff
@@ -250,7 +250,7 @@ place to store packages for multiple operating systems. Packages for both Ubuntu
       sudo sh -c 'echo "deb http://packages.elasticsearch.org/elasticsearch/1.7/debian stable main" >> /etc/apt/sources.list'
 
-At this point you will need to restart dashboard and mcp-server services
+   At this point you will need to restart dashboard and mcp-server services
```
Now that you're here! `s/mcp-server/archivematica-mcp-server`

sevein reviewed (approved):
Can this wait until #108 is merged? @sallain should be back from lunch soon, I can help her with her rebase (#110 came before).

jrwdunham committed:
Add firewall details and clarify post-install config

- Adds a firewall section to the Ubuntu packages installation page
  (install-ubuntu.rst.)
- Clarifies the "Post-install configuration" sections of the
  install-ubuntu.rst, install-centos.rst, and install-ansible.rst pages.
- Adds more details to the "Firewall" section of install-advanced.rst.

Addresses #954

sevein reviewed (approved):
Crystal clear!
